admin/src/scenes/phase0/components/CniField.jsx

```diff
@@ -38,6 +40,7 @@ export function CniField({
   const [requestButtonClass, setRequestButtonClass] = useState("");
   const [mouseIn, setMouseIn] = useState(false);
   const [cniModalOpened, setCniModalOpened] = useState(false);
+  const datePassed = young.latestCNIFileExpirationDate ? new Date(young.latestCNIFileExpirationDate) < new Date() : false;
```

Comment:
```suggestion
  const isExpired = young.latestCNIFileExpirationDate  ? isBefore(new Date(young.latestCNIFileExpirationDate), new Date()) : false
```
on a `date-fns` qui nous permet d'harmoniser la gestion des dates et que je trouve plus lisible, qu'en penses tu ?
`import { isBefore } from "date-fns";`
https://date-fns.org/v3.6.0/docs/isBefore

Reply:
ok je connaissais pas je l'ai mis